src/Razor/src/Microsoft.VisualStudio.Editor.Razor/ClassifiedSpan.cs

```diff
@@ -7,17 +7,14 @@ namespace Microsoft.VisualStudio.Editor.Razor;
 
 public struct ClassifiedSpan
 {
-    public ClassifiedSpan(SourceSpan span, SourceSpan blockSpan, SpanKind spanKind, BlockKind blockKind, AcceptedCharacters acceptedCharacters)
+    public ClassifiedSpan(SourceSpan span, SourceSpan blockSpan, SpanKind spanKind, BlockKind blockKind)
     {
         Span = span;
         BlockSpan = blockSpan;
         SpanKind = spanKind;
         BlockKind = blockKind;
-        AcceptedCharacters = acceptedCharacters;
```

Comment:
GetEditHandler is the source of this information, which is why I'm removing it here. It appears entirely used.